docs/changelog/pre-4.0.rst

```diff
@@ -988,7 +988,7 @@ Downloads
 -  `Official download
    page <https://www.powerdns.com/downloads>`__
 -  `native RHEL5/6 packages from Kees
-   Monshouwer <http://www.monshouwer.eu/download/3rd_party/pdns-server/>`__
+   Monshouwer <https://www.monshouwer.eu/download/3rd_party/pdns/>`__
```

Comment:
while I didn't generally canonicalize, I did for this one specific page which actively redirected (the site didn't redirect for recursor page)